ignite/templates/app/files/go.mod.plush

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/cometbft/cometbft v0.38.0
 	github.com/cosmos/cosmos-db v1.0.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.3
-	github.com/cosmos/cosmos-sdk v0.50.0-rc.1.0.20231005140444-10bd5a2cacdc
+	github.com/cosmos/cosmos-sdk v0.50.0-rc.1.0.20231018124024-3abc61fb75d4
```

Comment:
Is this a new version?

Reply:
I always try to use the latest commit. At the time it was.